lms/session.py

```diff
@@ -14,17 +14,18 @@ def includeme(config):
 
     config.set_session_factory(
         SignedCookieSessionFactory(
+            # https://docs.pylonsproject.org/projects/pyramid/en/latest/api/session.html#pyramid.session.SignedCookieSessionFactory
             secret=config.registry.settings["session_cookie_secret"],
             secure=secure,
             # ``httponly=True`` is recommended by the Pyramid docs to protect
-            # the cookie from cross-site scripting vulnerabilities, see:
-            # https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/sessions.html
```

Comment:
These links all point to the same place. Added a link to where all this params are declared at the top.